examples/bri-3/src/bri/identity/bpiSubjectAccounts/agents/bpiSubjectAccountsStorage.agent.ts

```diff
@@ -28,12 +28,12 @@ export class BpiSubjectAccountStorageAgent extends PrismaService {
     return new BpiSubjectAccount(
```

Comment:
now that the prisma model and domain model are in sync, we can directly invoke mapper(bpiSubjectAccountModel, BpiSubjectAccount,BpiSubjectAccount)